exercise_150507_Calculator.rb

```diff
@@ -0,0 +1,86 @@
+# File Name: 150507_Calculator_Exercise.rb
+# Developer: catchingash
+
+# Establish synonyms for math functions. NOTE: All text in the arrays needs to be lowercase.
+subtract = ["substraction", "sub", "subtract", "-"]
+add = ["addition", "add", "+"]
+multiply = ["multiplication", "multiply", "x", "*"]
+divide = ["division", "div", "/"]
+exponent = ["exponential", "exponents", "exponent", "exp", "power", "^", "**"]
+sqrt = ["squareroot", "square root", "sqrt", "^1/2", "\u221A"]
+mod = ["remainders", "remainder", "modulo", "modulus", "%"]
+operations = subtract + add + multiply + divide + exponent + sqrt + mod
+UNACCEPTED_OPERATION_MEMO = "I'm sorry. I'm confused. I know it's probably obvious to a human, but could you try again and dumb it down for me? (You'll need to restart me. Sorry!)"
+UNEXPECTED_ERROR_MEMO = "I'm not sure what happened. I unencountered an unexpected error and had to close. I'm sorry!"
```

Comment:
I like the extraction of constants into named variables at the top - it makes the rest of the code read much better.